openvdb/openvdb/python/pyOpenVDBModule.cc

```diff
@@ -326,13 +326,8 @@ struct VecTypeDescr
 ////////////////////////////////////////
 
 
-#ifdef DWA_OPENVDB
 #define PY_OPENVDB_MODULE_NAME  _openvdb
```

Comment:
Hmm, the last commit I made failed to build due to a name collision in `CMakeLists.txt` when I used `openvdb` as the package name here, which is why I used `_openvdb` as the module name here originally.

> Minor: If _openvdb was renamed to openvdb, the __init__.py file wouldn't be needed.

If we don't include an `__init__.py`, the import statement for the `openvdb` bindings looks like either

```python
from openvdb import _openvdb
...
_openvdb.foo(...)
```

Or 

```python
import openvdb._openvdb
...
openvdb._openvdb.foo(....)
```

Which is why I opted to include the `__init__.py` - it makes the `openvdb` site-packages directory a module. If this is really something that people want, I'm happy to make the change, though.

Reply:
Umm, that's curious. Probably because of the mismatch between the package name and the module name. Anyway, the `__init__.py` file doesn't hurt in any way.